src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -192,11 +192,11 @@ export class DictionaryModal extends React.Component {
                       value={data.public_access}
                     >
                       <option value="" />
-                      {(isEditingDictionary && dictionary.public_access === 'View') &&
-                      <option value="View" selected>Private </option>
+                      {(isEditingDictionary && dictionary.public_access === 'None') &&
+                      <option value="None" selected>Private </option>
                       }
-                      <option value="Edit">Public</option>
-                      <option value="View">Private</option>
+                      <option value="View">Public</option>
+                      <option value="None">Private</option>
```

Comment:
The display values 'Private' and 'Public' should match the values 'View' and 'None'